src/main/java/com/revature/RevRelay/controllers/GroupsController.java

```diff
@@ -50,7 +57,10 @@ public GroupsController(GroupService groupService) {
      * @return ResponseEntity<?> contains the response and the newly create group
      */
     @PostMapping
-    public ResponseEntity<?> createGroup(@RequestBody Group group) {
+    public ResponseEntity<?> createGroup(@RequestBody Group group, @RequestHeader("Authorization") String token) {
+        if (token != null && token.length() > 1) {
+            group.setUserOwner(userService.loadUserByUsername(jwtUtil.extractUsername(token.substring(7))));
```

Comment:
There's no need to use the JwtUtil to extract the username here, it's already done on the service layer.
```suggestion
            group.setUserOwner(userService.loadUserByToken(token.replace("Bearer", "").trim()));
```